mantidproject/mantid
# Plotting of specific lines in elemental analysis

This PR allows the user to select which lines to plot when visualizing a spectra in the `Elemental Analysis` interface (choice between `Total`, `Prompt`, `Delayed`).

**To test:**
 - `Interfaces->Muon->Elemental Analysis`
 - There should be three checkboxes, one for each line type
 - Load a run (eg `2695`)
 - Select different checkboxes and make sure the correct lines are plotted
 - Deselect them and ensure the correct lines are deleted
 - Delete lines from the remove line window and make sure the corresponding checkbox is unselected
 - When no box is selected you should not be able to plot anything


Fixes #26432.




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
